index.html

```diff
@@ -98,11 +96,15 @@ <h2 class="subSentenceText">I'm a Software Developer</h2>
     </section>
```

Comment:
✅ Good work on the implementation! I have a couple of suggestions to further align your code with the design:

1️⃣ For the "connectIcons" list that contains social icons, the left margin is currently set to 22px. However, according to the design, it should be 24px. Additionally, the gap between the icons should be 16px, but it is currently set to 24px. Please update your code to reflect these specifications and ensure visual consistency with the design.

2️⃣ The margin-top for the "littleParagraph" element is currently set to 14px. As per the design, it should be 12px. Please make the necessary adjustment in your code to match the design specifications.

By making these adjustments, your project will closely adhere to the design guidelines, providing a consistent and visually appealing user experience. Keep up the great work! 👏🎉